lib/utils/include/utils/graph/undirected.h

```diff
@@ -113,7 +114,10 @@ struct UndirectedGraph {
 
   UndirectedGraph &operator=(UndirectedGraph);
 
-  operator UndirectedGraphView() const;
+  operator UndirectedGraphView() const {
```

Comment:
See https://github.com/flexflow/FlexFlow/blob/6ef694251aea7b4c655f95cdbd4fb1e39e4bbcc5/lib/utils/src/graph/multidigraph.cc#L87-L89 for how to implement these